contracts/src/v0.8/l2ep/dev/optimism/OptimismValidator.sol

```diff
@@ -18,7 +16,9 @@ contract OptimismValidator is TypeAndVersionInterface, AggregatorValidatorInterf
   int256 private constant ANSWER_SEQ_OFFLINE = 1;
   uint32 private s_gasLimit;
 
+  // solhint-disable-next-line chainlink-solidity/prefix-immutable-variables-with-i
```

Comment:
I dont get this

Reply:
immutable variables should always start with `i_`. These don't. I just added an ignore line instead of changing it since its public and adding the i_ would change the ABI